library.js

```diff
@@ -417,6 +422,15 @@ async function filterPrivilegedUids (uids, cid, toPid) {
 	return uids.filter(Boolean);
 }
 
+async function filterDisallowedFullnames (users) {
+	users = await Promise.all(users.map(async (user) => {
+		const userSettings = await User.getSettings(user.uid);
```

Comment:
Can use `User.getMultipleUserSettings(uids);` instead to load all settings in one call.

Reply:
Thanks, made the change.